docs/DBAPI_COMPAT.md

```diff
@@ -1,8 +1,25 @@
 ## DB-API 2.0 compatibility
 
-PyEXASOL [public interface](/docs/REFERENCE.md) is similar to [PEP-249 DB-API 2.0](https://www.python.org/dev/peps/pep-0249/) specification, but it does not strictly follow it. This page explains the reasons behind this decision.
+PyEXASOL [public interface](/docs/REFERENCE.md) is similar to [PEP-249 DB-API 2.0](https://www.python.org/dev/peps/pep-0249/) specification, but it does not strictly follow it. This page explains the reasons behind this decision and your alternative(s) if you need or want to use a dbabpi2 compatible driver..
 
-If you absolutely need DB-API 2.0 compatibility, you may use [TurbODBC](https://github.com/blue-yonder/turbodbc) instead.
+### Alternatives
+
+#### Exasol WebSocket Driver
+The `pyexasol` package includes a DBAPI2 compatible driver facade, located in the `exasol.driver` package. However, using pyexasol directly will generally yield better performance when utilizing Exasol in an OLAP manner, which is likely the typical use case. 
+
+That said, in specific scenarios, the DBAPI2 API can be advantageous or even necessary. This is particularly true when integrating with "DB-Agnostic" frameworks. In such cases, you can just import and use the DBAPI2 compliant facade as illustrated in the example below.
+
+```python
+from exasol.driver.websocket.dbapi2 import connect
+
+connection = connect(dsn='', username='sys', password='exasol', schema='TEST')
+
+with connection.cursor() as cursor:
+    cursor.execute("SELECT 1;")
+```
+
+#### TurboODBC
```

Comment:
is it only turboodbc, or also pyodbc?

Reply:
pyodbc should work too I guess

Reply:
Should we add it?

Reply:
> pyodbc is an open source Python module that makes accessing ODBC databases simple. It implements the DB API 2.0 specification but is packed with even more ...
https://pypi.org/project/pyodbc/

it does, and yes please add it